src/Approximations/overapproximate.jl

```diff
@@ -1722,11 +1722,9 @@ continuous dynamics (Doctoral dissertation).
 """
 function _overapproximate_zonotope_cpa(X::LazySet, dir::Type{<:AbstractDirections})
     n = dim(X)
-
     # overapproximate 2D blocks
     if n > 1
-        nblocks = Int(floor(n/2))
-        πX_2D = [project(X, [i, i+1]) for i in 1:2:nblocks]
+        πX_2D = [project(X, [i, i+1]) for i in 1:2:n]
```

Comment:
Will this work for odd `n`? Is there such a test? If not, please add one.

Reply:
Yes, that is addressed here: https://github.com/JuliaReach/LazySets.jl/blob/1bee735828f19d0c167975f7fd9a2758a5811dfe/src/Approximations/overapproximate.jl#L1734

Reply:
But that comes only later.

```julia
julia> collect([i, i+1] for i in 1:2:9)
5-element Array{Array{Int64,1},1}:
 [1, 2]
 [3, 4]
 [5, 6]
 [7, 8]
 [9, 10]
```
If you do `project(X, [9, 10])` for a 9-dimensional set, I think this should crash.

Reply:
You are right, and there are tests with odd dimensionality, it errored  :)

Reply:
After your commit it works for odd `n`, but does it for even `n`?

Reply:
Yes! it should:
```julia
julia> collect([i, i+1] for i in 1:2:10)
5-element Array{Array{Int64,1},1}:
 [1, 2]
 [3, 4]
 [5, 6]
 [7, 8]
 [9, 10]
```

Reply:
But you are only going up to `n-1`, so for `n = 10` you are now doing `1:2:(10-1)`.

Reply:
Yes, I edited the comment (sorry) and it is the same result

Reply:
Oh indeed!